### PR TITLE
BayesianOptimization GSoC 2022

### PR DESCRIPTION
**Pull request by jbrea**

### CODE REVIEW

File: jsoc/gsoc/bayesopt.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# BayesianOptimization
+
+Bayesian optimization is a global optimization strategy for (potentially noisy) functions with unknown derivatives.
+With well-chosen priors, it can find optima with fewer function evaluations than alternatives, making it well suited for the optimization of costly objective functions.
+Well known examples include hyper-parameter tuning of machine learning models (see e.g. [Taking the Human Out of the Loop: A Review of BayeRecommended Skills:sian Optimization](https://www.cs.ox.ac.uk/people/nando.defreitas/publications/BayesOptLoop.pdf)).
```
Comment (aviks) [inline jsoc/gsoc/bayesopt.md:5]:
```suggestion
Well known examples include hyper-parameter tuning of machine learning models (see e.g. [Taking the Human Out of the Loop: A Review of Bayesian Optimization](https://www.cs.ox.ac.uk/people/nando.defreitas/publications/BayesOptLoop.pdf)).
```

**Comment by aviks** [comment]
Thanks. All projects this year need to specify if they are short (175 hours) or long (350 hours). There should also be an expected difficulty level.

**Comment by jbrea** [comment]
Thanks for the feedback, @aviks!

**Comment by logankilpatrick** [comment]
@jbrea if you create a new page, you also need to add a link on the main GSoC section so folks can find the projects

**Comment by jbrea** [comment]
Thanks, @logankilpatrick

**Comment by logankilpatrick** [comment]
Thank you!

**Pull request merged**